src/libutil/filter.cpp

```diff
@@ -289,8 +281,20 @@ class FilterBlackmanHarris1D : public Filter1D {
         const float A2 =  0.14128f;
         const float A3 = -0.01168f;
         const float m_pi = float (M_PI);
+#if 0
```

Comment:
If you nuke the line that does `x=(x+1)/2`, you can flip all the constants to be positive.